tests/parens/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -43,7 +43,9 @@ exports[`new.php 1`] = `
         new MessageFormatter(),
         'en',
         $someOtherVar,
-        array('foo' => 'bar')
+        array(
+            'foo' => 'bar'
+        )
```

Comment:
Also, the change to `hasNewlineInRange` seems to introduce this regression (I didn't see yet why)

Reply:
@czosel Very interesting, we should investigate why it is happens, because it can be parser bug and we should report about this

Reply:
Ah, I found the issue: 
```php
text = text.replace(/\?>\n<\?/g, "?>\n___PSEUDO_INLINE_PLACEHOLDER___<?");
```
is breaking the calculation of `offset`, because the placeholders are not contained in `options.originalText`.

I'll revert the implementation, so this issue won't affect this PR anymore. We should probably fix the offset/placeholder issue though?

Reply:
@czosel i think we can solve this only when parser will be create pseudo inline nodes, somebody can create issue in parser?

Reply:
Yeah, this should definitely be solved in the parser. I'll create an issue tomorrow!